notebooks/Model_Training_with_BERT.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>Can you add a computation of precision and recall at this point in the notebook, then redo the precision/recall computation after correcting for the BERT model splitting tokens? There should be a measurable increase in accuracy as a result of that correction.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Yeah, that's a good idea. It does give it a little bump ~0.85 -&gt; ~0.87 for F1. @frreiss do you happen to know why the model sometimes splits tokens that way - is it an expected artifact or is something wrong?</p>